openshift/templates/env_canada.cronjob.yaml

```diff
@@ -28,6 +28,7 @@ objects:
         cronjob: ${JOB_NAME}
     spec:
       schedule: "*/60 * * * *"
```

Comment:
Maybe every 30 mins would be better now that there is processing checks in place

Reply:
I'd still be concerned about how often it just runs into 404 errors (before the latest files get dropped)  - I'm ok with keeping it 60 minutes for now - especially that soon we'll be downloading even more data (when we start downloading the high resolution data)

Reply:
Okidoki